app/views/provisioning_templates/_custom_tabs.html.erb

```diff
@@ -3,7 +3,7 @@
 
   <div id="kind_selector" <%= display? @template.snippet %>>
     <%= select_f f, :template_kind_id, TemplateKind.all, :id, :to_s, {:include_blank => true, :selected => @template_kind_id},
-                 {:disabled => @template.locked?, :label => _("Type")} %>
+                 {:disabled => @template.locked?, :label => _("Type"), :label_help => _(f.object.template_kind.description)} %>
```

Comment:
Can you actually remove this one? It does not make sense for this kind of edit form, sorry my bad. Other than that it is good to merge.